docs/source/metrics.rst

```diff
@@ -144,43 +85,24 @@ Predefined Epoch Metrics
     :members:
 
 
-TorchMetrics
-------------
-
-`TorchMetrics <https://torchmetrics.readthedocs.io/>`__ is a library implementing a lot of known metrics in PyTorch.
-They are passed to :class:`~poutyne.Model` and :meth:`ModelBundle.from_network() <poutyne.ModelBundle.from_network()>` using either the  ``epoch_metrics`` argument or the ``torch_metrics`` argument.
-When passed as epoch metrics, they are only displayed at the end of the epoch.
-When passed using the specialized ``torch_metrics`` argument, they are displayed for each batch and at the end of the epoch.
-See `TorchMetrics documentation <https://torchmetrics.readthedocs.io/en/latest/references/modules.html>`__ for available torch metrics.
-
-.. warning:: When using the ``torch_metrics`` argument, the torch metrics are computed at each batch.
-    This can significantly slow down the computations depending on the metrics used.
-    In such cases, we advise using them as epoch metrics instead.
-
-Here is an example using TorchMetrics:
-
-.. code-block:: python
+Functional
+----------
 
-  from poutyne import Model
-  import torchmetrics
+Below is the functional version of some of the classes in the :ref:`object oriented metrics` section.
 
-  model = Model(
-      network,
-      'sgd',
-      'cross_entropy',
-      torch_metrics=[torchmetrics.F1(num_classes=10)],
-  )
-  model.fit_dataset(train_dataset, valid_dataset)
+.. autofunction:: acc
+.. autofunction:: bin_acc
+.. autofunction:: topk
 
 
 .. _multiple metrics at once:
 
 Computing Multiple Metrics at Once
 ----------------------------------
 
-When passing the metrics to :class:`~poutyne.Model` and :meth:`ModelBundle.from_network() <poutyne.ModelBundle.from_network()>`, the name of each batch and epoch metric can be change by passing a tuple ``(name, metric)`` instead of simply the metric function or object, where ``name`` is the alternative name of the metric.
+When passing the metrics to :class:`~poutyne.Model` and :meth:`ModelBundle.from_network() <poutyne.ModelBundle.from_network()>`, the name of each metric can be change by passing a tuple ``(name, metric)`` instead of simply the metric function or object, where ``name`` is the alternative name of the metric.
```

Comment:
```suggestion
When passing the metrics to :class:`~poutyne.Model` and :meth:`ModelBundle.from_network() <poutyne.ModelBundle.from_network()>`, each metric name can be changed by passing a tuple ``(name, metric)`` instead of simply the metric function or object, where ``name`` is the alternative name of the metric.
```